meson_options.txt

```diff
@@ -0,0 +1,11 @@
+# Feature options
+option('enable-tests', type : 'boolean', value : true, description : 'Enable tests')
+option('enable-check', type : 'boolean', value : true, description : 'Enable check tests, need the enable-tests flag set')
```

Comment:
Check tests? It's quite confusing what it does

Reply:
These are some tests that were already in the package, using a library called gst-check. This was also an option in the autotools version, so I added it here.